kapitan/inputs/helm.py

```diff
@@ -84,46 +99,80 @@ def compile_file(self, file_path, compile_path, ext_vars, **kwargs):
     def default_output_type(self):
         return None
 
-    def render_chart(self, chart_dir, output_path, **kwargs):
+    def render_chart(self, chart_dir, output_path, helm_params, helm_values_file, helm_values_files):
         args = ["template"]
-        """renders helm chart located at chart_dir, and stores the output to output_path"""
-        if kwargs.get("helm_values_files", []):
-            for file_name in kwargs["helm_values_files"]:
-                args.append("-f")
-                args.append(file_name)
 
-        if kwargs.get("helm_values_file", None):
-            args.append("-f")
-            args.append(kwargs["helm_values_file"])
+        name = helm_params.pop("name", None)
 
-        if kwargs.get("namespace", None):
-            args.append("-n")
-            args.append(kwargs["namespace"])
-
-        args.append("--output-dir")
-        args.append(output_path)
+        flags = {"--include-crds": True, "--skip-tests": True}
 
         if self.kube_version:
-            args.append("--api-versions")
-            args.append(self.kube_version)
+            flags["--api-versions"] = self.kube_version
+
+        for param, value in helm_params.items():
+            if len(param) == 1:
+                logger.warning("invalid helm flag: '%s'. helm_params supports only long flag names", param)
+                continue
+
+            param = param.replace("_", "-")
+
+            if param in ("set", "set-file", "set-string"):
+                logger.warning(
+                    "helm '%s' flag is not supported. Use 'helm_values' to specify template values", param
+                )
+                continue
+
+            if param == "values":
+                logger.warning(
+                    "helm '%s' flag is not supported. Use 'helm_values_files' to specify template values files",
+                    param,
+                )
+                continue
+
+            if param in HELM_DENIED_FLAGS:
+                logger.warning("helm flag '%s' is not supported and will be ignored.", param)
+                continue
```

Comment:
Should we maybe abort here? Warning output might be missed and lead to unexpected behavior if the flags are just dropped.